pkg/spanz/map.go

```diff
@@ -36,12 +37,19 @@ func lessSpanItem[T any](a, b spanItem[T]) bool {
 // Map is a specialized btree map that map a Span to a value.
 type Map[T any] struct {
 	tree *btree.BTreeG[spanItem[T]]
+
+	cache *struct {
+		coveredSpans, holes []tablepb.Span
+		lastGC              time.Time
+	}
 }
 
 // NewMap returns a new SpanMap.
 func NewMap[T any]() *Map[T] {
+	// Map is read heavy map, many Ascend.
+	const defaultDegree = 256
```

Comment:
Maybe it's better to move this constant variable out of `NewMap()`

Reply:
I file a task in https://github.com/pingcap/tiflow/issues/7720, the btree map will only be used in scheduler package which has to deal with a larger number of tables and lots of Ascend. I will address your comment in the task.